### DISCUSSION TITLE
How are the images generated in this docs?

### DISCUSSION THREAD

**Discussion by Gitstar-OC** [nisabmohd/Aria-Docs#6] [Q&A] [answered]
@nisabmohd, I am building a docs site for my project and I want to know which type of model are you using for the images static or dynamic (images will be rendered on server first), I am currently testing with the nextra docs and they are rendering the whole thing in a static way which I don't want. I want lazy loading for the images so that only the required images will be loaded. Switching to this template will be an easier option as I am in the prototype stage. Could you please tell me which type of model is used in this template for images or I will have to create a seperate component in nextra docs for lazy loading of the images. 

Happy Coding

**Comment by nisabmohd** [comment, marked as answer]
Hi, currently there is no strategy for handling images. You can use Next.js's Image component and add the loading prop to manage lazy loading or configure image handling according to your preferences. Additionally, you can pass custom components to the markdown parser to handle images or other elements as needed.

**Reply by Gitstar-OC** [reply]
thanks!!
